webtty/webtty.go

```diff
@@ -72,7 +72,12 @@ func (wt *WebTTY) Run(ctx context.Context) error {
 		errs <- func() error {
 			buffer := make([]byte, wt.bufferSize)
 			for {
-				n, err := wt.slave.Read(buffer)
+				//base64 length
+				effectiveBufferSize := wt.bufferSize - 1
+				//max raw data length
+				maxChunkSize := int(effectiveBufferSize/4) * 3
```

Comment:
Oh, that's a good catch!

For compatibility reasons, we don't always use base64 encoding, so this should depend on the encoder being used.